### DISCUSSION TITLE
A super small update request

### DISCUSSION THREAD

**Discussion by waeiski** [jpanther/lynx#35] [Ideas]
Hey thanks for this template and all your hard work. My first ever site is made with this template, I really love it. Thank you.

I have only a super small suggestion regarding Mastodon and its color. They changed their "brand color" to purple last spring (see here: https://blog.joinmastodon.org/2022/06/mastodon-branding-updates/ ). I am not entirely sure, but the purple appears to be this one #5D3FD3. Would it be possible to update the color to match?

Thank you.

**Comment by CallumKerson** [comment]
I have opened a PR to address to update the Mastodon colour. https://github.com/jpanther/lynx/pull/36

I went with `#6364ff` as that is what is used on the [Mastodon Blog CSS here](https://github.com/mastodon/blog/blob/master/static/style.css) and is also used in this PR for the simple icons project https://github.com/simple-icons/simple-icons/issues/7662
